config/application.rb

```diff
@@ -33,5 +33,12 @@ class Application < Rails::Application
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
+
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins "localhost:5000", "lvh.me:5000", "127.0.0.1:5000"
+        resource "*", headers: :any, methods: [:get, :post, :put, :patch, :delete]
```

Comment:
They send OPTIONS request too. I think that we can open all methods for CORS
```suggestion
        resource "*", headers: :any, methods: :any
```